temporalcli/commandsmd/commands.md

```diff
@@ -2521,12 +2534,40 @@ temporal workflow trace \
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
-### temporal workflow update: Start and wait for Updates (Experimental)
+### temporal workflow update: Updates (Experimental)
 An Update is a synchronous call to a Workflow Execution that can change its
 state, control its flow, and return a result.
 
 Experimental.
 
+#### Options set for update targeting
```

Comment:
The named options set should be under the first command that uses it instead of this command. This now shows that this command accepts these args. Can move and replace the `#### Options` section for `temporal workflow update describe` with this section.